presto-main/src/main/java/io/prestosql/sql/planner/PlanBuilder.java

```diff
@@ -115,7 +115,7 @@ public PlanBuilder appendProjections(Iterable<Expression> expressions, SymbolAll
         // add an identity projection for underlying plan
         projections.putIdentities(root.getOutputSymbols());
 
-        Map<ScopeAware<Expression>, Symbol> mappings = new HashMap<>();
+        Map<ScopeAware<Expression>, Symbol> mappings = new LinkedHashMap<>();
```

Comment:
Why?

Also, this change seems unrelated.

Reply:
It was needed at one point in the evolution of the implementation, but isn't any more.  Reverted.